config/config.go

```diff
@@ -29,6 +29,7 @@ type RawChainConfig struct {
 	Id       string            `json:"id"`       // ChainID
 	Endpoint string            `json:"endpoint"` // url for rpc endpoint
 	From     string            `json:"from"`     // address of key to use
+	Scan     bool              // should scan
```

Comment:
Follow the existing structure. Add json path.

Reply:
Updated